packages/flutter/test/widget/draggable_test.dart

```diff
@@ -753,4 +753,148 @@ void main() {
       expect(onDraggableCanceledOffset, equals(new Offset(flingStart.x, flingStart.y) + new Offset(0.0, 100.0)));
     });
   });
+
+  test('Drag and drop - allow pass thru of unaccepted data test', () {
+    testWidgets((WidgetTester tester) {
+      List<int> acceptedInts = <int>[];
+      List<double> acceptedDoubles = <double>[];
+
+      tester.pumpWidget(new MaterialApp(
+        routes: <String, WidgetBuilder>{
+          '/': (BuildContext context) { return new Column(
+            children: <Widget>[
+              new Draggable<int>(
+                data: 1,
+                child: new Text('IntSource'),
+                feedback: new Text('IntDragging')
+              ),
+              new Draggable<double>(
+                data: 1.0,
+                child: new Text('DoubleSource'),
+                feedback: new Text('DoubleDragging')
+              ),
+              new Stack(children:[
+                new DragTarget<int>(
+                  builder: (BuildContext context, List<int> data, List<dynamic> rejects) {
+                    return new IgnorePointer(
+                      child: new Container(
+                        height: 100.0,
+                        child: new Text('Target')
+                      )
+                    );
+                  },
+                  onAccept: (int data) {
+                    acceptedInts.add(data);
+                  }
+                ),
+                new DragTarget<double>(
+                  builder: (BuildContext context, List<double> data, List<dynamic> rejects) {
+                    return new IgnorePointer(
+                      child: new Container(
+                        height: 100.0,
+                        child: new Text('Target')
+                      )
+                    );
+                  },
+                  onAccept: (double data) {
+                    acceptedDoubles.add(data);
+                  }
+                ),
+              ])
+            ]);
+          },
+        }
+      ));
+
+      Point intLocation = tester.getCenter(tester.findText('IntSource'));
+      Point doubleLocation = tester.getCenter(tester.findText('DoubleSource'));
+      Point targetLocation = tester.getCenter(tester.findText('Target'));
+
+      // Drag the double draggable.
+      expect(acceptedInts, isEmpty);
+      expect(acceptedDoubles, isEmpty);
+      expect(tester.findText('IntSource'), isNotNull);
+      expect(tester.findText('IntDragging'), isNull);
+      expect(tester.findText('DoubleSource'), isNotNull);
+      expect(tester.findText('DoubleDragging'), isNull);
+      expect(tester.findText('Target'), isNotNull);
```

Comment:
so when you check this, you're only checking the first. which is probably fine, just mentioning this in case it wasn't your intent.